src/components/PatientProfilePage/PatientProfilePage.tsx

```diff
@@ -0,0 +1,448 @@
+import { useMutation } from '@apollo/react-hooks';
+import { useQuery } from 'react-apollo';
+import { useSnackbar } from 'notistack';
+import ConfirmModal from '../common/ConfirmModal';
+import { Colours } from '../../styles/Constants';
+import FormField from '../common/FormField';
+import CompletePatientButton from './CompletePatientButton';
+import RadioSelector from '../common/RadioSelector';
+import TriagePills from './TriagePills';
+import StatusPills from './StatusPills';
+import PatientTransportPage from './PatientTransportPage';
+import {
+  TriageLevel,
+  Status,
+  Patient,
+  Gender,
+  GET_PATIENT_BY_ID,
+  GET_ALL_PATIENTS,
+} from '../../graphql/queries/patients';
+import { ADD_PATIENT, EDIT_PATIENT } from '../../graphql/mutations/patients';
+import { Hospital, GET_ALL_HOSPITALS } from '../../graphql/queries/hospitals';
+import {
+  Ambulance,
+  GET_ALL_AMBULANCES,
+} from '../../graphql/queries/ambulances';
+import { CCP, GET_CPP_BY_ID } from '../../graphql/queries/ccps';
+
+interface FormFields {
+  barcodeValue: string;
+  triage: TriageLevel | null;
+  gender: Gender;
+  age: number | null;
+  notes: string;
+  runNumber?: number | null;
+  collectionPointId?: number;
+  status: Status | null;
+  triageCategory?: number | null;
+  TriageLevel?: number | null;
+}
+
+const PatientProfilePage = ({
+  match: {
+    params: { eventId, ccpId, patientId, barcodeValue },
+  },
+  mode,
+}: {
+  match: {
+    params: {
+      eventId: string;
+      patientId?: string;
+      ccpId: string;
+      barcodeValue?: string;
+    };
+  };
+  mode: string;
+}) => {
+  const history = useHistory();
+  const { enqueueSnackbar } = useSnackbar();
+  const [openTransportModal, setOpenTransportModal] = useState(false);
+  const [openTransportPage, setOpenTransportPage] = useState(false);
+  const [transportConfirmed, setTransportConfirmed] = useState(false);
+  const [transportingPatient, setTransportingPatient] = useState(false);
+  const [selectedHospital, setSelectedHospital] = useState<Hospital>({
+    id: '',
+    name: '',
+  });
+  const [selectedAmbulance, setSelectedAmbulance] = useState<Ambulance>({
+    id: '',
+    vehicleNumber: 0,
+  });
+
+  const { data, loading } = useQuery(
+    mode === 'edit' && patientId
+      ? GET_PATIENT_BY_ID(patientId)
+      : GET_ALL_PATIENTS
+  );
+  const patients: Array<Patient> = data ? data.patient : [];
+  const { data: hospitalData } = useQuery(GET_ALL_HOSPITALS);
+  const hospitals: Array<Hospital> = hospitalData ? hospitalData.hospitals : [];
+  const { data: ambulanceData } = useQuery(GET_ALL_AMBULANCES);
+  const ambulances: Array<Ambulance> = ambulanceData
+    ? ambulanceData.ambulances
+    : [];
+  const { data: ccpData } = useQuery(GET_CPP_BY_ID(ccpId));
+  const ccp: CCP = ccpData ? ccpData.collectionPoint : [];
+
+  const [addPatient] = useMutation(ADD_PATIENT, {
+    update(cache, { data: { addPatient } }) {
+      cache.writeQuery({
+        query: GET_ALL_PATIENTS,
```

Comment:
Right now, this line updates the `GET_ALL_PATIENTS` query results, but `CCPDashboardPage` uses `FETCH_ALL_PATIENTS`, so we don't see a new patient included without a page refresh. cc @petorsong 

Can you make these changes to fix this?
1. Update `CCPDashboardPage.tsx` to also use `GET_ALL_PATIENTS`
2. Make sure all fields are returned in the `GET_ALL_PATIENTS` query (I think only `status` is missing) 
3. and **remove @client** from `GET_ALL_PATIENTS`.

Reply:
I made the changes but I still don't see a new patient until I refresh the page, maybe this bug needs to be looked into further and should be part of another ticket?

Reply:
The suggestion I made here fixes the "bug" :) https://github.com/uwblueprint/paramedics-react/pull/27/files#r482688395

I am going to push one commit that makes the change I suggested to resolve the issue and removes `FETCH_ALL_PATIENTS` from the codebase completely. Let me know if you want the commit reverted.